src/Illuminate/Support/Testing/Fakes/PendingMailFake.php

```diff
@@ -21,18 +21,18 @@ public function __construct($mailer)
     /**
      * Send a new mailable message instance.
      *
-     * @param  \Illuminate\Contracts\Mail\Mailable  $mailable;
-     * @return mixed
+     * @param  \Illuminate\Contracts\Mail\Mailable  $mailable
+     * @return void
```

Comment:
Here I removed the extra semicolon, and also changed the return to match the interface.

Mailer implementation send method does not return anything.